helm/macropower-analytics-panel-server/Chart.yaml

```diff
@@ -9,5 +9,5 @@ sources:
 - https://github.com/MacroPower/macropower-analytics-panel/tree/master/server
 version: [[ .Version ]]
 annotations:
-  config.giantswarm.io/version: 1.x.x
+  config.giantswarm.io/version: "analytics-server-disable-user-metrics"
```

Comment:
I tried to test this with the [associated change in config](https://github.com/giantswarm/config/pull/345) but I couldn't get this to work. Regardless of whether the syntax is right in the deployment template, this `config.giantswarm.io/version` setting wouldn't be respected, i.e. config-controller generates the relevant ConfigMap from released version as it does not include the field added in [analytics-server-disable-user-metrics](https://github.com/giantswarm/config/tree/analytics-server-disable-user-metrics) branch.

Reply:
Looks like both the repo and the config branch have to be the same name. Closing this PR in favor of https://github.com/giantswarm/macropower-analytics-panel-server-app/pull/20

Reply:
OK, that's surprising, or maybe even a bug. Gotta ask you about it when you're back, feels like we should at least document it in https://intranet.giantswarm.io/docs/dev-and-releng/configuration-management/#how-can-i-test-giantswarmconfig-changes-without-making-a-release